src/main/java/com/epam/testsystem/github/dao/ApprovementDao.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.testsystem.github.dao;
+
+import com.epam.testsystem.github.enums.ApprovementStatus;
+import com.epam.testsystem.github.exception.BusinessLogicException;
+import com.epam.testsystem.github.model.Approvement;
+import lombok.RequiredArgsConstructor;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+import static com.epam.testsystem.github.dao.DaoExtractorUtil.APPROVEMENT_ROW_MAPPER;
+import static org.springframework.dao.support.DataAccessUtils.singleResult;
+
+@Repository
+@Transactional(propagation = Propagation.MANDATORY)
+@RequiredArgsConstructor
+public class ApprovementDao {
+    private final JdbcTemplate jdbcTemplate;
+
+    public Approvement add(final long userId, final long taskId) {
+        final LocalDateTime approveTime = LocalDateTime.now().withNano(0);
+        try {
+            jdbcTemplate.update(
+                    "INSERT INTO approvements (task_id, user_id, approve_time) " +
+                            "VALUES (?, ?, ?)",
+                    taskId, userId, approveTime);
+        } catch (final DataAccessException ignored){
+        }
+
+        return find(userId, taskId).orElseThrow(() -> new BusinessLogicException("Error in approvement add"));
```

Comment:
exception - my fault, ok.
But here i can't using mysql last_inserted_id, because, row have no id. And there is a situation, where no rows affected. I think normal - find by two id, which are unique key.
Bad way? Which? Use find?